### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by laniiiebb** [community/community#43503] [GitHub Education]
Bonjour, j'ai validé tout ce que je pouvais valider pour avoir le Student Developer Pack mais ça ne fonctionne pas. Ma carte étudiante n'est jamais acceptée je ne comprend pas pourquoi. On me demande de mettre mon profil à jour et c'est ce que j'ai fait...

**Comment by jongwooo** [comment]
@laniiiebb Please contact GitHub through the following [link](https://support.github.com/request/education?tags=dotcom-footer%2Chubberfy_education). I think GitHub staff will be happy to help.
